### ISSUE TITLE
Can Baritone break temporary blocks

### ISSUE THREAD

**Issue by xBRUHHY** [cabaletta/baritone#2976]
## What do you need help with?
When using #build and other features, Baritone can place temporary blocks if allowPlace is enabled, but is there a way to make baritone break the blocks after using them?

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by StylexTV** [comment]
No, right now there is no way for Bariton to behave like you asked, and I don't think Bariton will ever be able to behave like that, because if Bariton for example places 1 block in front of it and then uses that block to do a parkour jump, then there is no way to go back and destroy that temporary block (ok, breaking it while flying through the air might be possible, but I don't know, that sounds very impractical).

**Comment by xBRUHHY** [comment]
theres something called mlg

**Comment by StylexTV** [comment]
Yes, I am aware of mlgs, short for Major League Gaming, which describe the act of placing a water bucket right before hitting the ground in order to prevent fall damage. What I don't know is how they relate to your question.

**Comment by xBRUHHY** [comment]
you said there way no way to go back

**Comment by StylexTV** [comment]
Yeah, that's correct but I still don't get what mlgs have to do with breaking blocks.

**Comment by ZacSharp** [comment]
I think the problem of MLGs and long parkour jumps (which Baritone can't do anyway) can be avoided by making the option incompatible with `allowPlace`, just like `backfill` is incompatible with `allowBreak`.

Theoretically (don't think it is likely to happen) it would also be possible to disallow doing those movements starting on a scaffolding block. Placing the block is independent of the movement used to reach the position so it should not break the invariant of movements only being dependent on the position they start from. (Unless I'm wrong) it also does not break other invariants so it can be done without breaking optimality.
